css/extensible-search-typeahead.css

```diff
@@ -0,0 +1,9 @@
+.esp-overlay {
+    position: absolute;
+    background-color: white;
+    border: 1px solid;
+    border-top: 0;
+    padding: 1em;
+    display: none;
+	z-index: 9001; // ;)
```

Comment:
On the SilverStripe Simple theme if added to the header, it will end up under the site menu by default. These are really basic and ugly defaults intended to get it displaying on a page for designers to play with.